src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,31 @@
 
 #include "t04_both2.h"
 #include <iostream>
-
-using namespace std;
+#include <set>
 
 int t04_both2() {
+	int n = 0, m = 0, x = 0;
+	std::multiset<int> mst;
+
+	std::cin >> n;
+
+	for (int i = 0; i < n; ++i) {
+		std::cin >> x;
+		mst.insert(x);
+	}
+
+	std::cin >> m;
+
+	for (int i = 0; i < m; ++i) {
+		std::cin >> x;
+		mst.insert(x);
+	}
+
+	for (std::multiset<int>::iterator i = mst.begin(); i != mst.end(); ++i)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing bounds check for extra iterator increment in loop.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4511665)